Textile/Textile.sublime-syntax

```diff
@@ -1,225 +1,1119 @@
-        - meta_scope: markup.other.table.textile
-        - match: ^$
-          captures:
-            1: entity.name.tag.footnote.textile
-            3: entity.name.type.textile
-            4: entity.name.tag.footnote.textile
-          pop: true
-        - include: inline
-        - include: scope:text.html.basic
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.footnote.textile
+      push: textile-table-body
+
+  textile-table-body:
+    - meta_scope: markup.other.table.textile
+    - include: textile-empty-line-pop
+    - include: html
+    - include: textile-inlines
+
+  textile-inline-table-row:
+    - match: '^\|({{textile_attributes_optional_class_style}})(\\\||.)+\|$'
+      scope: markup.other.table.cell.textile
+      captures:
+        1: entity.name.type.textile
+
+###[ BLOCKS / PARAGRAPHS ]#####################################################
+
+  # https://textile-lang.com/doc/paragraphs
+
+  textile-paragraphs:
+    - match: |-
+        (?x:
+          ^
+          (p)
+          ({{textile_indentation_optional}})
+          ({{textile_alignment_optional}})
+          ({{textile_attributes_optional_class_style}})
+          (\.\.)
+          [ ](?=\S)
+        )
+      scope: entity.name.section.paragraph.textile
+      captures:
+        1: entity.name.tag.paragraph.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.paragraph.textile
+      push: textile-extended-paragraph-body
+    - match: |-
+        (?x:
+          ^
+          (p)
+          ({{textile_indentation_optional}})
+          ({{textile_alignment_optional}})
+          ({{textile_attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
+      scope: entity.name.section.paragraph.textile
+      captures:
+        1: entity.name.tag.paragraph.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.paragraph.textile
+      push: textile-paragraph-body
+    - match: ^(?=\S)
+      push: textile-paragraph-body
+
+  textile-extended-paragraph-body:
+    - meta_scope: meta.paragraph.textile
+    - include: textile-next-block-pop
+    - include: html
+    - include: textile-inlines
+
+  textile-paragraph-body:
+    - meta_scope: meta.paragraph.textile
+    - include: textile-empty-line-pop
+    - include: html
+    - include: textile-inlines
+
+###[ BLOCKS / LISTS ]##########################################################
+
+  textile-lists:
+    - match: '^\*'
+      captures:
+        1: markup.list.unnumbered.textile
+      push: textile-unordered-list-block
+
+    - match: '^\#'
+      captures:
+        1: markup.list.numbered.textile
+      push: textile-ordered-list-block
+
+    # TODO: definition lists
```

Comment:
Any plans for those TODOs?

Reply:
I am currently just in code-style and architecture-cleaning mode. Feel free to work on those Textile related features.